README.md

```diff
@@ -290,21 +300,70 @@ schedule:
 
 # job-pipeline
 
-A pipeline job allows combining multiple jobs together into a single job which can be configured to run
-the jobs in series.  Supporting parallel execution is planned but not yet supported.
+A pipeline job allows combining multiple jobs together into a single job.  The execution of these jobs is configurable by an optional
+"execution" element.  If not specified, the default execution settings are:
+
+```yaml
+execution:
+  maxConcurrentJobs: 1
+  maxRetriesPerJob: 0
+  retryInterval: 5
+  retryIntervalUnit: "MINUTES"
+```
+
+Jobs can either be specified inline or by referencing a separate job.yml file as shown below.
 
 Example configuration:
 
 ```yaml
 type: "job-pipeline"
 configuration:
   jobs:
-    - "load-upper-neno.yml"
-    - "load-lower-neno.yml"
+    - path: "load-upper-neno.yml"
+    - type: "job-pipeline"
+      configuration:
+        jobs:
+          - "load-lower-neno.yml"
```

Comment:
Maybe a super nit, but probably worth breaking inline and referenced jobs because this is a little confusing as-is (but is a rare use case)... on first read it could be interpreted that "path" and "type" apply to the same job.  Just thinking about others getting confused when reading the docs.

Reply:
Not a strong preference, just a thought